modules/ROOT/pages/spring-data-neo4j.adoc

```diff
@@ -1,318 +1,478 @@
-image::{img}sdn.png[float="right"]
+[#reactive-development]
+== Reactive Development
 
-Neo4j offers a rich set of possibilities for developers using Spring.
-If you are looking to use Neo4j on the JVM in general, the link:../java[Java developer's guide] explains from that perspective.
-However, if you want to benefit from full-fledged object mapping and the other helpful support that comes with Spring Data, this guide shows how to do that with *Spring Data Neo4j*.
+Neo4j (version 4.0+) incorporated the principles of the https://www.reactivemanifesto.org/[reactive manifesto^] for passing data between the database and client with the drivers.
+Developers can take advantage of the reactive approach to process queries and return results.
+This means that communication between the driver, and the database can be managed and adjusted dynamically according to data needs of the client.
 
-Neo4j's Spring Data integration was the founding project of the Spring Data efforts, started by Rod Johnson and Emil Eifrem.
-It is built on http://spring.io/[Spring Framework^] and also offers Object-Graph Mapping (OGM) on top of Neo4j.
+Reactive programming principles allow the consuming side (applications and other systems) to specify the amount of data received within a certain window of time.
+Neo4j's database driver will also maintain rate limits for requesting data from the server, providing flow control throughout the entire Neo4j stack.
 
-The link:../neo4j-ogm[Neo4j-OGM^] provides a plain, Java object-graph mapper and integrates in the http://projects.spring.io/spring-data[Spring Data^] infrastructure, including Spring Data repository and annotated object-mapping support.
+No matter the volume of transactions or data (even during times of high activity), the system can maintain limits on how much it can send and receive at once based on available resources.
+This prevents overloads and collapses or failures, as well as lost transmissions or later catch up loads during the downtime.
 
-Spring Data Neo4j is also supported by Spring Boot.
-You can use the https://start.spring.io/[Spring Initializr page^] to get started with Spring Data Neo4j immediately.
+https://projectreactor.io/[Project Reactor^] is the core foundation of many implementations of reactive development, including https://spring.io/reactive[Spring's^].
+Neo4j uses the Spring implementation of Project Reactor components to provide reactive support in related applications with the graph database.
 
-[#spring-data]
+[#sdn-reactive]
 == Spring Data Neo4j
 
-Though Spring Data Neo4j has been around for a long time, Neo4j's APIs and usage evolved quickly from an embedded, Java-only database to a server solution with mostly link:/developer/cypher-query-language/[Cypher] interactions.
+The Spring Data Neo4j 6 is the new major version of the Spring Data Neo4j project.
+Its feature benefit is the capability and support for reactive transactions, though there are other improvements and additions, as well.
```

Comment:
```suggestion
One of its feature benefit is the capability and support for reactive transactions, though there are other improvements and additions, as well.
```
Yes, the following example is reactive based but I don't see reactive support as the outstanding feature. I could imagine to have even _full immutable support_ / _record based mapping_ in this paragraph.

Reply:
immutability and record mentioned in fc88b3aa4bbae2a0dc163d11141ccee2a1dfc259

Reply:
+ missing word added in 828f631 😅